### PR TITLE
Added Nette framework

### PR DESCRIPTION
**Pull request by lulco**

### CODE REVIEW

File: php/nette/composer.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "require": {
+        "nette/application": "^3.0",
```
Comment (waghanza) [inline php/nette/composer.json:3]: Could you use `~3.0.0`  ? We need to stick to **3.0**, the `^` notation could change the major

File: php/nette/composer.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        "nette/bootstrap": "^3.0",
```
Comment (waghanza) [inline php/nette/composer.json:4]: Same as above

File: php/nette/composer.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+        "nette/di": "^3.0"
```
Comment (waghanza) [inline php/nette/composer.json:5]: Same as above

File: php/nette/composer.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    },
+    "autoload": {
```
Comment (waghanza) [inline php/nette/composer.json:7]: What is this for ?

**Comment by waghanza** [comment]
@dg Are you Ok (at least not opposed) to include `nette` in this list ?

**Comment by dg** [comment]
> @dg Are you Ok (at least not opposed) to include `nette` in this list ?

Yes, of course :)

File: php/nette/composer.json (hunk 1 above)
Comment (lulco) [reply php/nette/composer.json:3]: sure, I will change

File: php/nette/composer.json (hunk 4 above)
Comment (lulco) [reply php/nette/composer.json:7]: this is for loading all classes in `app` directory using composer, I am pretty sure that application will not work without it

Comment (waghanza) [reply php/nette/composer.json:7]: ah, it is not related to `composer autoload` (which is done in `Dockerfile`)

it was probably a noob question, I am more a `rubyist` that a `php` adopter

**Comment by lulco** [comment]
Thanks for merge! How often results are updated?

**Pull request merged**